internal/pkg/cli/deploy/backend.go

```diff
@@ -126,20 +126,35 @@ func (d *backendSvcDeployer) validateALBRuntime() error {
 	if d.backendMft.RoutingRule.IsEmpty() {
 		return nil
 	}
+	if err := d.validateALBRuntimeFor(d.backendMft.RoutingRule.Main); err != nil {
+		return fmt.Errorf("validate ALB runtime configuration for http: %w", err)
+	}
+	for idx, rule := range d.backendMft.RoutingRule.AdditionalRoutingRules {
+		if err := d.validateALBRuntimeFor(rule); err != nil {
+			return fmt.Errorf("validate ALB runtime configuration for http.additional_rules[%d]: %w", idx, err)
```

Comment:
```suggestion
			return fmt.Errorf(`validate ALB runtime configuration for "http.additional_rules[%d]": %w`, idx, err)
```
also can we have a test case covering this error msg?

Reply:
Yeah we have it [here](https://github.com/aws/copilot-cli/pull/4650/files#diff-abcea5573166990296031a09372d242d7a1a7ff625e48ff67526725f37525614R219)